src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -19,14 +19,10 @@ public class ConnectionUtil {
         }
     }
 
-    public static Connection getConnection() {
+    public static Connection getConnection() throws SQLException {
```

Comment:
better leave as it was, so we can have information specific to bad connection, not just general can't do something